server/src/main/scala/com/cloudera/livy/server/LivyServer.scala

```diff
@@ -122,6 +122,18 @@ class LivyServer extends Logging {
       server.context.addFilter(csrfHolder, "/*", EnumSet.allOf(classOf[DispatcherType]))
     }
 
+    if (livyConf.getBoolean(ACCESS_CONTROL_ENABLED)) {
+      if (livyConf.get(AUTH_TYPE) == KerberosAuthenticationHandler.TYPE) {
```

Comment:
Should be enough to check that it's not `null`, in case we ever add another auth type.